### PR TITLE
Refactored User Commands

### PR DESCRIPTION
**Pull request by skosman**
- Removed the user command file and restructured the commands into their own components.
- Changed the file structure to match the drawing Dianna did so components are separated. So now have transaction service with all the commands and quote server stuff, plus database in its own directory.
- Updated the getLog to include quote server, account transactions and got rid of extra admin user error.
- Also created triggers handler but it hasn't been incorporated yet

### CODE REVIEW

File: backend/audit/logXML.py [hunk 1]
```diff
@@ -85,7 +85,7 @@ def logSystemEvent(infoDict):
     if 'filename' in infoDict:
         ElementTree.SubElement(systemEvent, 'filename').text = infoDict['filename']
     if 'amount' in infoDict:
-        ElementTree.SubElement(systemEvent, 'funds').text = infoDict['amount']
+        ElementTree.SubElement(systemEvent, 'funds').text = '%.2f' % infoDict['amount']
 
     prettyPrintLog(root)
```
Comment (DamonAknuh) [inline backend/audit/logXML.py]: prettyPrintLog statement can be scrubed here, and probably removed in other functions aswell?

File: database/src/database.py [hunk 2]
```diff
@@ -1,29 +1,44 @@
...
+        """
+
         try:
             client = pymongo.MongoClient(HOST, DB_PORT)
             Database.DATABASE = client[DB_NAME]
             print("-----CONNECTED TO MONGODB DB-----")
-        
+            Database.DATABASE[ACCOUNTS_COLLECT].drop()
+            Database.DATABASE[TRANSACT_COLLECT].drop()
+            Database.DATABASE[TRIGGER_COLLECT].drop()
```
Comment (DamonAknuh) [inline database/src/database.py:35]: Just something to watch out here is that each time a webserver is started the database will be cleared.

Which is probably fine.

**Review by DamonAknuh — APPROVED** [review]

File: backend/audit/logXML.py (hunk 1 above)
Comment (DiannaBanana) [reply backend/audit/logXML.py:90]: I agree, I think we can just remove all the instances of prettyPrint except where it actually needs to be called

Comment (DiannaBanana) [reply backend/audit/logXML.py]: I agree, I think we can just remove all the instances of prettyPrint except where it actually needs to be called

File: transaction_service/src/quoteServer/triggers_handler.py [hunk 3]
```diff
@@ -0,0 +1,166 @@
+"""
+    Consistenly calls the quote server and triggers collection to execute any buy/sell triggers
+"""
+"""
+    TODO:
+        - add caching to limit amount of calls to quote server
+        - be incorporated into transaction server or organize it so it can be run independently
+"""
```
Comment (DiannaBanana) [inline transaction_service/src/quoteServer/triggers_handler.py:8]: Nice! Will probably need your help to set up our trigger handlers with the Django API

**Review by DiannaBanana — APPROVED** [review]

File: backend/audit/logXML.py [hunk 4]
```diff
@@ -20,7 +20,7 @@
 from xml.etree import ElementTree, cElementTree
 from xml.dom import minidom
 
-auditNum = "1"
+auditNum = "45"
```
Comment (DanielZenk) [inline backend/audit/logXML.py]: Can we add some kind of command line statement so we don't have to go into this file and change it every time? Not a big concern for this pull request, just some super minor thing to think about.

**Review by DanielZenk — APPROVED** [review]

File: backend/audit/logXML.py (hunk 1 above)
Comment (skosman) [reply backend/audit/logXML.py]: Yup, I'll remove it!

File: transaction_service/src/quoteServer/triggers_handler.py (hunk 3 above)
Comment (skosman) [reply transaction_service/src/quoteServer/triggers_handler.py:8]: Sounds good!

File: backend/audit/logXML.py (hunk 4 above)
Comment (skosman) [reply backend/audit/logXML.py]: Yup! I'll try to add one real quick.

**Pull request merged**